middlewares.go

```diff
@@ -64,11 +64,11 @@ func JwtMiddleware(key string) negroni.Handler {
 // Cors middleware
 func Cors(origin []string, headers []string) negroni.Handler {
 	return negroni.HandlerFunc(func(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+		w.Header().Set(headerAllowOrigin, strings.Join(origin, ","))
 		if r.Method == "OPTIONS" && r.Header.Get("Access-Control-Request-Method") != "" {
 			w.Header().Set(headerAllowMethods, strings.Join(defaultAllowMethods, ","))
 			w.Header().Set(headerAllowHeaders, strings.Join(headers, ","))
 			w.Header().Set(headerAllowCredentials, strconv.FormatBool(true))
```

Comment:
On our VueJS app the `headerAllowCredentials` is also missing.

Can you add this header too?